doc/manual/var_t_return.md

```diff
@@ -22,3 +22,13 @@ then it has to be derived from the routine's first parameter:
 In other words, the lifetime of what `result` points to is attached to the
 lifetime of the first parameter and that is enough knowledge to verify
 memory safety at the call site.
+
+When a `var`-typed return value is assigned to a local variable using a
```

Comment:
What happens when you assign to a global `var` in the global root namespace?.

I would mention or link `out` parameters too.

Reply:
local was a misnomer, changed to `var`-declared value